src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -526,6 +584,7 @@ public void startedGesture(String nameOfGesture) {
     if (gestureAlreadyStarted) {
       warn("Warning 1 gesture already running, this can break spacetime and lot of things");
     } else {
+      log.info("Starting gesture : {}", nameOfGesture);
       gestureAlreadyStarted = true;
       RobotCanMoveRandom = false;
```

Comment:
variable names should start with a lower case letter by java conventions.

Reply:
agree, that's why I started to rename some.. there is a risk to break stuff for now